10_date_timestamps_and_lies.md

```diff
@@ -145,31 +143,10 @@ df.select(to_date($"lastUpdate").as("lastUpdate"))
 // +----------+-----+
 ```
 
-and
-
-```scala
-df.select(to_date($"lastUpdate").as("lastUpdate"))
-   .groupBy($"lastUpdate").count
-   .orderBy($"lastUpdate").printSchema
-```
-
-gives the following:
-
-```scala
-// root
-//  |-- lastUpdate: date (nullable = false)
-//  |-- count: long (nullable = false)
-```
-
 whereas,
 
 ```scala
-df.select(to_date($"lastUpdate").as("lastUpdate"))
-  .groupBy($"lastUpdate")
-  .count
-  .orderBy($"lastUpdate")
-  .filter($"lastUpdate".isNull)
-  .show
+df.filter($"lastUpdate".isNull).show
```

Comment:
Nice way to illustrate that would be to show that it is neither `"null"`:

    df.filter($"lastUpdate" === "null").show

nor

    df.filter($"lastUpdate".isNull).count

but in fact there are `nulls`:

    df.rdd.map(_.getTimestamp(0)).filter(_ == null).count